modules/ROOT/pages/showcase-apps/identity-management.adoc

```diff
@@ -1,2 +1,23 @@
 include::{partialsdir}/attributes.adoc[]
 
+= {keycloak-service}
+
+[discrete]
+== Prerequisites
+
+* [x] You have setup the services as described in xref:showcase-apps/service-setup.adoc[Service Setup].
```

Comment:
Shouldn't this link to the specific keycloak setup guide?

Reply:
I thought linking to service setup for showcase apps is a better idea. That document has the links to Keycloak setup guide and also some additional steps (user creation) for the showcase app.
What do you think?